### PR TITLE
Updated PyJWT Dependency

### PR DESCRIPTION
**Pull request by clintonb**
- Using PyJWT 0.4.1 (or newer)
- Relying on PyJWT to verify ID token audience and issuer

Replaces #486.

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @dsjen @jbau @dylanrhodes

File: requirements-python3.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ requests>=1.1.0
 oauthlib>=0.3.8
 requests-oauthlib>=0.3.0,<0.3.2
 six>=1.2.0
-PyJWT==0.2.1
+PyJWT>=0.4.1
```
Comment (Menda) [inline requirements-python3.txt]: I would use `==` for safety.

Comment (clintonb) [inline requirements-python3.txt]: Good point. `>=` is what lead to this problem in the first place.

**Comment by Menda** [comment]
Restart also Travis, it's failing for any reason.

**Comment by omab** [comment]
Thanks!

**Pull request merged**